rero_ils/modules/patrons/jsonschemas/patrons/patron-v0.0.1.json

```diff
@@ -417,7 +423,19 @@
         "expressionProperties": {
           "templateOptions.required": "field.parent.model.roles.some(v => (v === 'librarian' || v === 'system_librarian'))"
         },
-        "fieldMap": "library"
+        "fieldMap": "libraries",
+        "validation": {
+          "validators": {
+            "uniqueValueKeysInObject": {
+              "keys": [
+                "$ref"
+              ]
+            }
+          },
+          "messages": {
+            "uniqueValueKeysInObjectMessage": "Only one library is allowed"
```

Comment:
The message is not correct. You can belong to multiple libraries, but duplicate values are not allowed in this list

Reply:
done